test/typedarray/sort.js

```diff
@@ -1,12 +1,9 @@
 //-------------------------------------------------------------------------------------------------------
 // Copyright (C) Microsoft. All rights reserved.
-// Copyright (c) 2022 ChakraCore Project Contributors. All rights reserved.
+// Copyright (c) ChakraCore Project Contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
-// @ts-check

```

Comment:
Is this an annotation of some sort?

Reply:
It's an instruction to VsCode to run a typescript based typecheck on a javascript file, it does no harm but most of our test files don't have them.